lib/v2/screens/settings/components/settings_card.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/providers/services/navigation_service.dart';
+import 'package:seeds/v2/components/divider_jungle.dart';
+
+/// SETTINGS CARD
+class SettingsCard extends StatelessWidget {
+  /// Card icon
+  final IconData icon;
+
+  /// The text title in the first row
+  final String title;
+
+  /// The text value next to the title
+  final String titleValue;
+
+  /// To set a different color to titleValue
+  final Color titleValueColor;
+
+  /// The descrption text in the second row
+  final String descriptionText;
+
+  /// Use if the second row is a widget
+  final Widget descriptionWidget;
+
+  /// Route to which the card navigates.
+  final String route;
+
+  const SettingsCard({
+    Key key,
+    @required this.icon,
+    @required this.title,
+    @required this.titleValue,
+    this.titleValueColor,
+    this.descriptionText = '',
+    this.descriptionWidget,
+    this.route,
+  })  : assert(icon != null),
+        assert(title != null),
+        assert(titleValue != null),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 8.0),
+      child: InkWell(
+        onTap: () => NavigationService.of(context).navigateTo(route),
+        child: Container(
+          decoration: BoxDecoration(
+            color: AppColors.greenfield,
+            borderRadius: BorderRadius.circular(12.0),
```

Comment:
use the constant card radius  that i asked you to define. 

Make sure to use it in all the places where it applies